tests/test_vrf_af.rb

```diff
@@ -118,7 +132,7 @@ def route_target(af)
     refute(v.default_route_target_both_auto_evpn,
            'default value for route target both auto evpn should be false')
 
-    if platform == :ios_xr
+    if platform == :ios_xr || node.product_id[/N3/]
```

Comment:
Please update these to use the `validate_property_excluded?(feature, property)` api.

https://github.com/cisco/cisco-network-node-utils/blob/develop/tests/test_vrf.rb#L78